src/main/java/de/tum/in/www1/artemis/web/rest/StudentExamResource.java

```diff
@@ -227,9 +227,6 @@ public StudentExamResource(ExamAccessService examAccessService, StudentExamServi
         examSession.hideDetails();
         studentExam.setExamSessions(Set.of(examSession));
 
-        // not needed
-        studentExam.getExam().setCourse(null);
```

Comment:
For the complaint functionality we need the course in order to determine if the course allows for complaints